src/cat.ts

```diff
@@ -68,13 +74,22 @@ export class Cat {
 
     this.minTheta = minTheta;
     this.maxTheta = maxTheta;
-    this.prior = prior;
+    this.priorDist = priorDist;
+    this.priorPar = priorPar;
     this._zetas = [];
     this._resps = [];
     this._theta = theta;
     this._seMeasurement = Number.MAX_VALUE;
     this.nStartItems = nStartItems;
     this._rng = randomSeed === null ? seedrandom() : seedrandom(randomSeed);
+    Cat.validatePrior(priorDist, priorPar, minTheta, maxTheta);
+    if (priorDist === 'norm') {
+      this._prior = normal(priorPar[0], priorPar[1], minTheta, maxTheta);
+    } else if (priorDist === 'unif') {
+      this._prior = uniform(priorPar[0], priorPar[1], 0.1, minTheta, maxTheta);
+    } else {
+      this._prior = normal(0, 1, minTheta, maxTheta); // fallback
+    }
```

Comment:
I think this fallback is unnecessary sine we require that `priorDist` is either `norm` or `unif` in `Cat.validatePrior`.
```suggestion
    }
```